docs/integrations/sources/facebook-marketing.inapp.md

```diff
@@ -4,23 +4,36 @@
 
 ## Setup guide
 
+### For Airbyte Cloud:
+
+**To set up Facebook Marketing as a source in Airbyte Cloud:**
+
+1. [Log into your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account.
+2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ New source**.
+3. On the Set up the source page, select **Facebook Marketing** from the **Source type** dropdown.
+4. For Name, enter a name for the Facebook Marketing connector.
+
+**Facebook Marketing Source Settings:**
+
 1. Click **Authenticate your account** to authorize your [Meta for Developers](https://developers.facebook.com/) account. Airbyte will authenticate the account you are already logged in to. Make sure you are logged into the right account.
+2. Account ID: [Facebook Ad Account ID Number](https://www.facebook.com/business/help/1492627900875762): Open your Meta Ads Manager. Ad account ID number in the account dropdown menu or in your browser's address bar.
```

Comment:
Please update if you like the suggestion I made in the `spec.json` for this field.